controller/controller_server/csi_controller_server.py

```diff
@@ -658,6 +668,22 @@ def start_server(self):
             controller_server.stop(0)
             logger.debug('Controller Server Stopped ...')
 
+    def _get_source_type_and_id(self, request):
+        source = request.volume_content_source
+        object_id = None
+        source_type = None
+        if source:
+            logger.info(source)
+            if source.HasField(config.VOLUME_SOURCE_SNAPSHOT):
+                source_id = source.snapshot.snapshot_id
+                source_type = config.VOLUME_SOURCE_SNAPSHOT
+            else:
+                source_id = source.volume.volume_id
+                source_type = config.VOLUME_SOURCE_VOLUME
+            logger.debug("src_id_field: {0}".format(source_id))
```

Comment:
`src_id_field` should be `source_id`

Reply:
done

Reply:
in addition, we probably don't need both this and `logger.debug("Source {0} id : {1}".format(source_type, source_id))`, so choose which place to keep, and delete in the other (preferably logging both type and id)

Reply:
done